sdk/python/pvsite_datamodel/sqlmodels.py

```diff
@@ -74,6 +74,9 @@ class GenerationSQL(Base, CreatedMixin):
     """
 
     __tablename__ = "generation"
+    __table_args__ = (
+        UniqueConstraint("site_uuid", "start_utc", "end_utc", name="uniq_cons_site_start_end"),
```

Comment:
Should we go a step further and make the constraint on `(site_uuid, start_utc)`?

Reply:
I thought there might be funny situations where we have 5 minis values and 1 hour values. Or something like that. Then being really preciate with all three is useful

Reply:
Ok sure.